platform/windows/detect.py

```diff
@@ -710,8 +711,18 @@ def configure_mingw(env):
             )
             sys.exit(255)
 
+        env.AppendUnique(CPPDEFINES=["D3D12_ENABLED", "RD_ENABLED"])
+        env.Append(LIBS=["d3d12", "dxgi", "dxguid"])
+
         # PIX
-        if env["pix_path"] != "" and os.path.exists(env["pix_path"]):
+        if (
+            env["use_pix"]
+            and (env["arch"] == "x86_64" or env["arch"] == "arm64")
```

Comment:
And here :)

Or we could do the checks in `configure` and override `use_pix` to False to simplify the 4 checks.